unicef/giga-data-ingestion
# Feat/approval request db

## What type of PR is this?

- `feat`: Commits that add a new feature

## Summary

- Add approval request fixture yaml
- Add approval request fixture generation script
- Add models
- Update UI and API to use enabled states
- 
- Update entrypoint scripts
## How to test

1. Run `task` to initiate migration and loading of fixtures. You can also manually trigger these by running `task migrate` or `task load-fixtures approval_requests`. You may also cd into the api folder and run the commands directly: `poetry run alembic upgrade head`, `poetry run python -m scripts.load_fixtures approval_requests`
2. Go through the upload flow if you still don't have any approval requests generated
3. Assert that the approval requests have loading states,
4. Assert that after approval, the country-dataset you approved is blurred out.
5. You may use a tool like dbeaver or psql to inspect the db to verify that the enabled flag was unset.

## Link to Jira/Asana/Airtable task (if applicable)

https://app.asana.com/0/1206270489214055/1207112736836243

## Implementation screenshot/screencap (if applicable)


https://github.com/unicef/giga-data-ingestion/assets/122899250/37c5954b-5937-474f-968e-0b9c6c101000
